### ISSUE TITLE
Scaling CTC loss when length_average = True has no effects

### ISSUE THREAD

**Issue by weedwind** [SeanNaren/deepspeech.pytorch#319]
Hi, I found something strange. I used CTC loss to do phoneme recognition on TIMIT data. At first, I turned off length_average, so loss = CTCloss(), and I got good result. Then, I multiplied the loss by 0 in the forward pass, and the network training was not able to make progress, as expected. However, if I turned on length_average = True, whether multiplying the loss by 0 or not does not make a difference for the results, they all produce normal results (~80% phone accuracy on the test data). I used Adam optimizer, and I did not use any gradient clipping. Anyone has any ideas?

**Comment by SeanNaren** [comment]
Again duplicate of #250 going to address there @weedwind :)
